MyApplication3/app/getAndroidID.ps1

```diff
@@ -1,7 +1,17 @@
-cd C:\Users\camil\AppData\Local\Android\Sdk\platform-tools

```

Comment:
Can you move this script to the root folder - we are not using MyApplication3 anymore

Reply:
Completed